FeedAPIChallenge/Feed API/FeedsImageMapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FeedsImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Edil Ashimov on 7/29/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final internal class FeedImageMapper {
+	private static var OK_200: Int { return 200 }
+
+	private struct Root: Decodable {
+		let items: [Image]
+		var feed: [FeedImage] {
+			return items.map { $0.item }
+		}
+	}
+
+	private struct Image: Decodable {
+		public let id: UUID
+		public let description: String?
+		public let location: String?
+		public let url: URL
```

Comment:
`Image` is a private detail and a private struct. `public` modifiers will take no effect, please remove them

```suggestion
		let id: UUID
		let description: String?
		let location: String?
		let url: URL
```

Reply:
Thanks!Got it!